src/accordion/accordion.ts

```diff
@@ -115,14 +115,17 @@ export interface NgbPanelChangeEvent {
     <ng-template ngFor let-panel [ngForOf]="panels">
       <div class="card">
         <div role="tab" id="{{panel.id}}-header" [class]="'card-header ' + (panel.type ? 'bg-'+panel.type: type ? 'bg-'+type : '')">
-          <a href (click)="!!toggle(panel.id)" [class.text-muted]="panel.disabled" [attr.tabindex]="(panel.disabled ? '-1' : null)"
-            [attr.aria-expanded]="panel.isOpen" [attr.aria-controls]="(panel.isOpen ? panel.id : null)"
-            [attr.aria-disabled]="panel.disabled">
-            {{panel.title}}<ng-template [ngTemplateOutlet]="panel.titleTpl?.templateRef"></ng-template>
-          </a>
+          <h5 class="mb-0">
+            <button class="btn btn-link" (click)="!!toggle(panel.id)" [disabled]="panel.disabled" [class.collapsed]="panel.isOpen"
+              [attr.tabindex]="(panel.disabled ? '-1' : null)"
```

Comment:
I don't think we need this anymore if we use `<button>` - disabling a button should take it out of the normal tab-press focus change flow.